### DISCUSSION TITLE
[Question] How to remove "not submitted" assignments in KLMS?

### DISCUSSION THREAD

**Discussion by tncks0121** [prosyslab-classroom/cs348-information-security#693] [Q&A]
Name: Suchan Park

Sorry for uploading a non-course-related question here.

In the KLMS main page, there are some badges indicating how much VODs or assignments are left, as below.

<img width="673" alt="image" src="https://user-images.githubusercontent.com/3483637/230867398-dbefdefd-556d-4692-9686-0033fd3d90af.png">

<img width="127" alt="image" src="https://user-images.githubusercontent.com/3483637/230867216-c607f66c-c742-48e4-9f33-d742a16232c5.png">

It seems the assignments from this course are considered "not submitted", and thus in the badge.

<img width="377" alt="image" src="https://user-images.githubusercontent.com/3483637/230867235-fa33023c-3343-4d5f-8977-212d7f094244.png">

I skimmed through the [manual](https://klms.kaist.ac.kr/local/manual/view.php?id=1), and I cannot find a way to discard it. Is there a known way to remove the assignments from the "not submitted" count? It's not important at all, but quite distracts me, so I'm curious if there is a known solution for this.

**Comment by KihongHeo** [comment]
I don't know. I don't use KLMS.

**Comment by KAIST-JongchanPark** [comment]
@tncks0121 I just set a deadline for HW 01 and HW 02. Are those assignments removed from "not submitted"?

**Comment by tncks0121** [comment]
Yes, they are both removed. Thank you very much :)
